docker/README.md

```diff
@@ -6,6 +6,8 @@ You can use this when you want to run bbc-a11y from a machine on which you can't
 
 For example, electron is not easy to install on CentOS, so if you are using CentOS build agents in your continuous integration environment, this may be useful to you.
 
+If you're usng this in CI, then you may not need the docker image. For example in TravisCI you can prepend the run script with xvfb-run. Furthermore and alternative to using the docker image in Jenkins might be to use the [xvfb plugin](https://wiki.jenkins.io/display/JENKINS/Xvfb+Plugin) - though this is untested.
```

Comment:
There is a spelling error, a missing "i" in "usng". However, it may be worth rewording the first sentence.

"The docker image is not always necessary to use bbc-a11y for continuous integration. For example, ..."

Also, "Furthermore and ..." should be "Furthermore, an"